client/a8c-for-agencies/sections/marketplace/context.ts

```diff
@@ -0,0 +1,9 @@
+import { createContext } from 'react';
+import type { ShoppingCartContext as ShoppingCartContextInterface } from './types';
+
+export const ShoppingCartContext = createContext< ShoppingCartContextInterface >( {
+	selectedItems: [],
```

Comment:
`Items` is too generic. Can we name it as `CartItems`?

Reply:
Good point 👍